apps/angular/simple-animations/src/app/app.component.ts

```diff
@@ -19,7 +21,7 @@ import { Component } from '@angular/core';
   `,
   template: `
     <div class="mx-20 my-40 flex gap-5">
-      <section>
+      <section [@slideIn]>
```

Comment:
Are these brackets needed because you pass an array `ANIMATIONS`? Nice way of adding the animations by the way, i've always imported the animations seperate. This seems a bit cleaner 👍

Reply:
hey nice catch, thanks! I was binding it to some value but decided not to. But forgot to remove the binding syntax.
I have fixed that.